api/geth_backend.go

```diff
@@ -564,6 +564,19 @@ func (b *GethStatusBackend) ChangeDatabasePassword(keyUID string, password strin
 	return nil
 }
 
+func (b *GethStatusBackend) tryToDeleteAccount(address types.Address, password, newPassword string) error {
```

Comment:
@rasom once you know about the final requests for mobile app, if the only way to remove convert an account to a keycard account is from within the app (after user logs in) we should remove this and just use `b.accountManager.DeleteAccount` where is needed.

Reply:
yep I understand this, thanks